web/styles/rendered_markdown.css

```diff
@@ -987,3 +987,74 @@
         padding-left: 0.3em;
     }
 }
+
+.muted_sender {
+    font-style: normal;
+    font-weight: normal;
+    width: auto;
+    opacity: 0.5;
+    padding-bottom: 0;
+}
+
+.muted_sender_text {
+    font-style: normal;
+    font-weight: normal;
+    font-size: small;
+    width: auto;
+    opacity: 0.5;
+    padding-bottom: 0;
+}
+
+@media (width <= 600px) {
+    .muted_sender_text {
+        visibility: hidden;
+        overflow: hidden;
+    }
+}
+
+.reveal_button {
+    font-style: normal;
+    font-weight: 500;
+    color: var(--color-text-neutral-borderless-action-button);
+    text-decoration: none;
+}
+
+.reveal_button a {
+    color: inherit;
+    text-decoration: none;
+}
+
+.reveal_button a:hover,
+.reveal_button a:focus {
+    color: inherit;
+    text-decoration: none;
+}
```

Comment:
Add a class to the `a` element you want to reference -- this sort of CSS rule where you just have `a` at the end creates performance issues, and also isn't robust to changes in the HTML.